src/core/inlines.js

```diff
@@ -76,17 +76,23 @@ const inlineElement = /(?:\[\^[^^]+\^\])/; // Inline [^element^]
  */
 function inlineElementMatches(matched) {
   const value = matched.slice(2, -2).trim();
-  const [element, attribute, attrValue] = value
+  const [forPart, attribute, attrValue] = value
```

Comment:
Might be misleading to call it `forPart`? Not sure.

Reply:
element is more misleading for the global case, which is why I renamed it. It was weird otherwise. 

Eventually, we will refractor all this... we should have a single parser for all these types.